### ISSUE TITLE
Change term adapter to driver in documentation

### ISSUE THREAD

**Issue by MartinMuzatko** [alexsasharegan/vue-transmit#66]
Since the last commit (https://github.com/alexsasharegan/vue-transmit/commit/e00cef204aaa40cdfad1d3a8fedeafb8acd6d2d0)  the term has changed to driver.
The documentation still points to "adapter".

**Comment by alexsasharegan** [comment]
True! It's on my list, but the project is on hold for me while I'm deep in a react project. I'd welcome a PR for converting terminology!

**Comment by MartinMuzatko** [comment]
Will do
